docs/UserGuide.md

```diff
@@ -384,6 +385,9 @@ Alternatively, you can use csv config files to customize the analysis as before
 * `repo-config.csv`: Add additional rows for the extra repos ([example](repo-config.csv))
 * `author-config.csv`: Add one row for each author in each repo you want to analyze
 
+## Using RepoSense with Netlify
+
+You can use Netlify to run RepoSense with your configured files. Follow this [guide](UsingNetlifyGuide.md) to use RepoSense with Netlify.
```

Comment:
I think a reference in getting started to this section is needed to maximize its value.

Also, the description need to be more critical. Otherwise, people will misinterpret this section as a Netlify-specific component/functionality; people who don't use Netlify will simply ignores it.